File: comms/push-notifications/fetch-push-notification-operation/fetch-push-notification-operation.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;

$client = new TwilioComms(
    accountId: 'TWILIO_ACCOUNT_SID',
    authToken: 'TWILIO_AUTH_TOKEN',
);
$client->pushNotifications->fetchOperation(
    'comms_operation_01h9krwprkeee8fzqspvwy6nq8',
);

File: comms/push-notifications/fetch-push-notification-operation/fetch-push-notification-operation.java
package com.example.usage;

import com.twilio.sdk.TwilioComms;

public class Example {
    public static void main(String[] args) {
        TwilioComms client = TwilioComms
            .builder()
            .credentials("TWILIO_ACCOUNT_SID", "TWILIO_AUTH_TOKEN")
            .build();

        client.pushNotifications().fetchOperation("comms_operation_01h9krwprkeee8fzqspvwy6nq8");
    }
}